whois-api/src/main/java/net/ripe/db/whois/api/httpserver/JettyBootstrap.java

```diff
@@ -122,6 +126,7 @@ public JettyBootstrap(final RemoteAddressFilter remoteAddressFilter,
                           @Value("${http.idle.timeout.sec:60}") final int idleTimeout,
                           @Value("${http.sni.host.check:true}") final boolean sniHostCheck,
                           @Value("${dos.filter.enabled:false}") final boolean dosFilterEnabled,
+                          @Value("${dos.filter.max.updates:10}") final String dosUpdatesMax,
```

Comment:
is this per second? rename method if possible to specify

Reply:
yes, it is per second.
Ok I will rename the variable

Reply:
Done and moved to UpdateDoSFilterHolder